app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js

```diff
@@ -221,13 +197,19 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
 
     function updateLoadBalancerNames() {
       var account = $scope.loadBalancer.credentials,
-        region = $scope.loadBalancer.region;
+          region = $scope.loadBalancer.region;
 
-      if (allLoadBalancerNames[account] && allLoadBalancerNames[account][region]) {
-        $scope.existingLoadBalancerNames = allLoadBalancerNames[account][region];
-      } else {
-        $scope.existingLoadBalancerNames = [];
-      }
+      const accountLoadBalancersByRegion = {};
+      application.refresh(true).then(() => {
```

Comment:
I would change this to `application.getDataSource('loadBalancers').refresh(true)` - no need to refresh the entire application.